cmflib/merger.py

```diff
@@ -1,73 +1,292 @@
+###
+# Copyright (2022) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
 from cmflib import cmf
 import json
 
-
-def parse_json_to_mlmd(mlmd_json):
+# mlmd is created from metadata passed in Json format
+def parse_json_to_mlmd(mlmd_json, path_to_store, cmd):
     mlmd_data = json.loads(mlmd_json)
-    type(mlmd_data)
+    # type(mlmd_data)
     pipelines = mlmd_data["Pipeline"]
-    print(type(pipelines))
+    # print(type(pipelines))
     pipeline = pipelines[0]
```

Comment:
This way we are restricting to a single pipeline. Although we expect only a single pipeline, still we can put a for loop here and make the code more generic to iterate through all the pipelines that are received